app/addons/documents/mango/components/ExecutionStats.js

```diff
@@ -37,18 +35,10 @@ export default class ExecutionStats extends React.Component {
     return [minutes, ' ', minuteText, ', ', seconds, ' ', secondsText].join('');
   }
 
-  getWarning(executionStats, warning) {
-    if (!executionStats) { return warning; }
-
-    // warn if many documents scanned in relation to results returned
-    if (!warning && executionStats.results_returned) {
-      const docsExamined = executionStats.total_docs_examined || executionStats.total_quorum_docs_examined;
-      if (docsExamined / executionStats.results_returned > 10) {
-        return TOO_MANY_DOCS_SCANNED_WARNING;
-      }
+  getWarning(warning) {
+    if (warning) {
+      return warning.split('\n').join('<br>');
```

Comment:
This will print `<br>` instead of adding a line break. You can use this instead:
```
getWarning(warnings) {
    if (warnings) {
      return (
        <span>
          {warnings.split('\n').map(warnText => <>{warnText}<br/></>)}
        </span>
      );
    }
  }
```

Reply:
thanks @Antonio-Maranhao - updated with those changes